### DISCUSSION TITLE
Equipment Advice

### DISCUSSION THREAD

**Discussion by nebhead** [nebhead/PiFire#97] [Show and tell]
**Thought I would start a section here in the discussion area about general pellet grill equipment.  Feel free to share your thoughts on different pellet grills brands, models, components, accessories, etc.!**  

_I'll kick things off by sharing a couple of items that I have recently picked up that make life easier for my pellet grilling._ 

1. **Replacement bucket liners:**  If you have a Traeger (or other brand of pellet grill) with the bucket hanging off the side, you've probably struggled with lining it with tin-foil at least once.  Getting foil in the bucket is challenging and can lead to holes which allows the grease to get to the bottom of the bucket - ending up in a big mess to clean.  I found [these](https://smile.amazon.com/gp/product/B07ZWH2RRZ) disposable foil buckets on Amazon which fit perfectly into the Traeger bucket.   

![image](https://user-images.githubusercontent.com/19876886/166842634-fc6b6180-b1a7-4d30-92d9-768dbd717714.png)

2. **Pellet Storage:** I hated having to fumble around with 20lb or 40lb bags of pellets, attempting to pour them into the hopper without spilling pellets here and there.  [This pellet dispenser](https://smile.amazon.com/gp/product/B08BJDXBWB) has been perfect for me, holding a full bag of pellets and even allowing to keep the bag inside so you can remember what type they are.  It's got a couple options for pouring and is pretty air tight.  It's been awesome.  

![image](https://user-images.githubusercontent.com/19876886/166842839-547c85a4-9aab-4e83-8f86-f53490338398.png)

**Comment by nebhead** [comment]
FYI - I found [these Probe Spools](https://www.thingiverse.com/thing:2313618) on Thingiverse the other day.  I'm going to print them up and try them out to see if they are helpful.  

![image](https://user-images.githubusercontent.com/19876886/168429424-3140e154-3cce-418d-8bb0-0e7b9d39c40d.png)

**Reply by faisalhusain007** [reply]
These are awesome, I use these for most of my probes

**Reply by nebhead** [reply]
Printed it yesterday and it seems pretty nice.  My only addition would be some magnets, so that I can stick it to the side of the pellet hopper.

**Comment by niget2002** [comment]
I'm new to the group. Just ordered my parts to build a PiFire. I currently use a Fireboard on my pit boss for monitoring all the temperatures. It's old and some of the temp probe inputs no longer work, which is why I went looking for a replacement.

Some of the equipment that I use...

I really like their silicone pockets for storing probes:
https://www.fireboard.com/shop/probe-pouches/

I use Vittle Vault dog food buckets for storing pellets. You can get them in various sizes depending on how much you're willing to lift. You can get them on amazon or at local Tractor Supply stores if you have one nearby.

https://a.co/d/dth5Yzr

**Comment by nebhead** [comment]
> I'm new to the group. Just ordered my parts to build a PiFire. I currently use a Fireboard on my pit boss for monitoring all the temperatures. It's old and some of the temp probe inputs no longer work, which is why I went looking for a replacement.

Welcome to the community!  (BTW - We have a pretty good group of folks at the Discord  [link  in the readme] with some great conversation there)

> Some of the equipment that I use...
> 
> I really like their silicone pockets for storing probes: https://www.fireboard.com/shop/probe-pouches/

Very cool!  that gives me an idea to make this 3D printed (maybe with TPU).  

> I use Vittle Vault dog food buckets for storing pellets. You can get them in various sizes depending on how much you're willing to lift. You can get them on amazon or at local Tractor Supply stores if you have one nearby.
> 
> https://a.co/d/dth5Yzr

I never thought of this!  Super cool idea!
